server/app/views/components/QuestionSortOption.java

```diff
@@ -9,7 +9,7 @@
 public enum QuestionSortOption {
```

Comment:
Just a thought: should there be multiple enums for different types of sorting, such as QuestionSortOption and TIGroupSortOption?

For instance, I think sorting questions by "TI name" would be an error, so the code shouldn't allow that.

Reply:
I think putting QuestionSortOption and TiSortOptions as separate enums, as you suggested, is a good long term plan. In the short term, I'm OK with this PR. I will defer to the more experienced CiviForm developers, who may think otherwise. (I don't have a strong opinion on this.)